WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -476,7 +476,7 @@ - (void)setPost:(ReaderPost *)post
 
     _post = post;
 
-    if (_post.isWPCom) {
+    if (_post.isWPCom || _post.isJetpack) {
```

Comment:
I was looking at the [check to show the comment bubble on the list](https://github.com/wordpress-mobile/WordPress-iOS/blob/f79b84efc6ffa624a0e3f26088380c19dad10c7a/WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift#L436), and I think we need a couple extra checks:

- `_post.commentsOpen`
- `[AccountHelper isDotcomAvailable]`

This does fix the issue, so feel free to leave those for another PR

Reply:
Well the thing is, since those checks are done to either show or not the bubble comment if they don't pass we wouldn't reach this screen, right?

Reply:
Right, and I just realized `ReaderCommentsViewController` does check these two to decide if it should show the option to reply. Existing comments should load even if comments are closed 🤦‍♂️